### PR TITLE
Add embedded YouTube to README

### LINKED ISSUE #2266: Add new OpenCost video to README!
Proposal is to make the new OpenCost demo video with expanded scope more visible on the main repo's README! 

https://github.com/opencost/opencost/blob/develop/README.md

### PR DESCRIPTION
**Pull request by mattray**
Closes https://github.com/opencost/opencost/issues/2266

### CODE REVIEW

**Comment by dwbrown2** [comment]
Can we also update the README to reference new cloud cost support? Without that the video seems out of place IMO.

File: README.md [hunk 1]
```diff
@@ -30,6 +30,12 @@ Visit the full documentation for [recommended install options](https://www.openc
 - [Prometheus Metrics](https://www.opencost.io/docs/integrations/prometheus)
 - Reference [User Interface](https://github.com/opencost/opencost/tree/develop/ui)
 
+## Using the UI
```
Comment (dwbrown2) [inline README.md]: I recommend replacing the image at the top with this video?

**Comment by mattray** [comment]
@dwbrown2 added more cloud cost details, put the video up top, updated the screenshot, and added UI access instructions.

File: README.md [hunk 2]
```diff
@@ -30,6 +31,18 @@ Visit the full documentation for [recommended install options](https://www.openc
...
+
+The OpenCost UI provides a visualization of the Kubernetes allocations and the related Cloud Costs.
+
```
Comment (dwbrown2) [inline README.md]: Is this the best place for this info? Is there not a Usage page/guide that we can add to?

**Review by dwbrown2 — COMMENTED** [review]
Overall looks good, just one comment

File: README.md (hunk 2 above)
Comment (mattray) [reply README.md]: it's also in the [UI README](https://github.com/opencost/opencost/tree/develop/ui#readme) but not everyone's going to leave the base README

Comment (mattray) [reply README.md]: @dwbrown2 merge?

Comment (dwbrown2) [reply README.md]: Can we add a "Usage" section to docs nav and move this later? If so, I'm good to merge.

Comment (dwbrown2) [reply README.md]: @mattray updates? planning to remove this?

**Review by cliffcolvin — APPROVED** [review]

**Comment by dwbrown2** [comment]
@mattray still have a couple open comments in our slack thread regarding typos, etc. Can you address before merging?

**Comment by mattray** [comment]
@dwbrown2 I just pushed them

**Review by dwbrown2 — APPROVED** [review]

**Pull request merged**